index.html

```diff
@@ -48,35 +48,46 @@
 
           <div class="subscribe-form">
             <h2>Coming to Seattle and Berlin in 2016</h2>
-            <p class="stay-tuned">Stay tuned for more info!</p>
 
-<!--
-             <form>
+             <form id="newsletter-signup" name="newsletter-signup">
               <fieldset>
                 <legend><span>Subscribe for updates</span></legend>
                 <ol>
-                  <li>
-                    <label for="subscribe-email">E-mail</label>
-                    <input type="email" id="subscribe-email" name="email" required placeholder="you@example.com">
+                  <li class="error"></li>
+                  <li id="li_email">
+                    <label for="id_email">E-mail
+                      <input type="email" id="newsletter-email" name="email" required placeholder="you@example.com">
+                    </label>
+                  </li>
+                  <li id="li_newsletters" class="radio">
+                    <legend>I want to hear about...</legend>
+                    <label for="view-source-conference-north-america">
+                      <input type="radio" name="newsletters" value="view-source-conference-north-america" required />
+                      Seattle
+                    </label>
+                    <label for="view-source-conference-global">
```

Comment:
As the radio buttons don't have an ID, clicking the label text doesn't select the related radio. Small UI thing, but nice to have and easy to remedy.